hr_holidays_working_time/models/hr_holidays.py

```diff
@@ -55,7 +55,8 @@ def _auto_init(self, cr, context=None):
     number_of_hours = fields.Float(
         'Number of Hours', compute='_compute_number_of_hours', store=True)
     number_of_days_temp = fields.Float(
-        compute='_compute_number_of_hours_from_hours', store=True)
+        compute='_compute_number_of_hours_from_hours', store=True,
+        required=False)
```

Comment:
So, the field on the view is display as required field ?
> Concerning the constraint is it really necessary as the computing result can not be driven into a False computed value by the added algorithm?
Ok, so it's not necessary